gojira.sh

```diff
@@ -355,19 +361,19 @@ function get_envs {
 function create_kong {
   [[ $GOJIRA_MODE == "image" ]] && return
 
-  mkdir -p $GOJIRA_KONGS
-  pushd $GOJIRA_KONGS
-    local $remote
-    if [[ "$GOJIRA_GIT_HTTPS" = 1 ]]; then
-      remote="https://github.com/kong"
-    else
-      remote="git@github.com:kong"
-    fi
-    git clone -b ${GOJIRA_TAG} $remote/$GOJIRA_REPO.git $PREFIX || {
-      git clone $remote/$GOJIRA_REPO.git $PREFIX
-      pushd $PREFIX;
-        git checkout $GOJIRA_TAG || exit
-      popd
+  mkdir -p $GOJIRA_KONGS/$PREFIX
+
+  local url=$GOJIRA_GIT_REMOTE/$GOJIRA_REPO.git
+
+  pushd $GOJIRA_KONGS/$PREFIX
+    # clone a branch / tag
+    git clone -b ${GOJIRA_TAG} $url $PWD || {
+      # checkout SHA
+      git clone -n $url $PWD
+      git checkout $GOJIRA_TAG
+    } || {
+      rm -rf $GOJIRA_KONGS/$PREFIX
+      err "[!] could not clone $url ($GOJIRA_TAG)"
     }
   popd
```

Comment:
WDYT about using the subshell for changing directories? I know right now we are not using `set -eo pipefail`, but maybe in the future we do and this will at least ensure the starting directory is returned to.

```suggestion
  (
    cd $GOJIRA_KONGS/$PREFIX
    # clone a branch / tag
    git clone -b ${GOJIRA_TAG} $url $PWD || {
      # checkout SHA
      git clone -n $url $PWD
      git checkout $GOJIRA_TAG
    } || {
      rm -rf $GOJIRA_KONGS/$PREFIX
      err "[!] could not clone $url ($GOJIRA_TAG)"
    }
  )
```